### DISCUSSION TITLE
pygimli version 1.3.1?

### DISCUSSION THREAD

**Discussion by Jonathan-Nyquist** [gimli-org/pyGIMLi#482] [Q&A]
The documentation says to install version 1.3.1
```
conda create -n pg -c gimli -c conda-forge pygimli=1.3.1
```
But the repository shows 1.3.0 as the latest version released. Furthermore, if you just install without specifying a version you get version 1.2.6.  So what is the version we should be installing?

**Comment by halbmy** [comment]
Version v1.3.1 is perfect. We have no idea why conda prefers v1.2.6 (sometimes even 1.1.1 depending on your conda installation) over any 1.3 if the version is not specified. `mamba` (instead of `conda`, needs to be installed by conda first) does a better job and chooses 1.3.1 in my case.

We did not yet "draft" an official release (with release notes etc.) as we still have no MacOS build for pgcore v1.3, but that's on the todo list.

**Comment by Jonathan-Nyquist** [comment]
Thanks, Thomas. I work mostly on a Mac.  Any idea when the pgcore build will drift to the top of your TODO list? 🙂
________________________________
From: Thomas Günther ***@***.***>
Sent: Thursday, January 19, 2023 12:50 PM
To: gimli-org/gimli ***@***.***>
Cc: Jonathan Nyquist ***@***.***>; Author ***@***.***>
Subject: [External] Re: [gimli-org/gimli] pygimli version 1.3.1? (Discussion #482)


Version v1.3.1 is perfect. We have no idea why conda prefers v1.2.6 (sometimes even 1.1.1 depending on your conda installation) over any 1.3 if the version is not specified. mamba (instead of conda, needs to be installed by conda first) does a better job and chooses 1.3.1 in my case.

We did not yet "draft" an official release (with release notes etc.) as we still have no MacOS build for pgcore v1.3, but that's on the todo list.

**Reply by halbmy** [reply]
It is on the top of the list already but we were facing linking problems and have only limited access to Mac. We'll keep you updated.

**Comment by halbmy** [comment]
Outdated.
